frontend/factory-ops/src/component/DeleteItemModa.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FC, useState } from 'react';
+import Modal from 'react-bootstrap/Modal';
+import { Button } from 'react-bootstrap';
+
+interface DeleteItemProperties {
+	onDelete: () => void;
+}
+
+const DeleteItemModal: FC<DeleteItemProperties> = (props: DeleteItemProperties) => {
+	const [showModal, setShowModal] = useState<boolean>(false);
+
+	const handleClose = () => setShowModal(false);
+
+	const handleDelete = () => {
+		props.onDelete();
+		handleClose();
+	};
+
+	return (
+		<>
+			<Button variant="danger" size={'sm'} onClick={() => setShowModal(true)}>
+				Delete
+			</Button>
+			<Modal show={showModal} onHide={handleClose}>
+				<Modal.Header closeButton>
+					<Modal.Title>Confirm Deletion</Modal.Title>
+				</Modal.Header>
+				<Modal.Body>Are you sure you want to delete this item?</Modal.Body>
```

Comment:
It will be nice to see which item you want to delete, such as name, length, etc.

Reply:
added